### ISSUE TITLE
[media-kit] Autogen scanning software `media-gfx/xsane`

### ISSUE THREAD

**Issue by cuantar** [macaroni-os/mark-issues#93]
The ebuild for `xsane` needs maintenance.  We should autogen this package and can get the source from Github here: https://github.com/nphilipp/xsane

**Comment by geaaru** [comment]
A working workaround is to use `media-gfx/simple-scan`. The Gnome application alternative of `xsane`.

**Comment by geaaru** [comment]
It seems this the upstream right website: https://gitlab.com/sane-project/frontend/xsane

**Comment by jiggie42** [comment]
xsane installed for me, but I don't have a scanner device to test atm.  @cuantar are you still having a problem with it?

I compared our `xsane-0.999-r4` to gentoo's `-r6` ebuild and the only additional fix they have is for `clang16` 
```$ xsane -v
xsane-0.999 (c) 1998-2013 Oliver Rauch

This package is modified from the original version.
Please contact your vendor or report problems at
http://bugs.gentoo.org

  E-mail: Oliver.Rauch@xsane.org
  package xsane-0.999
  compiled with GTK-2.24.32
  without GIMP support
  XSane output formats: jpeg, pdf(compr.), png, pnm, ps(compr.), tiff, txt```

Ideally we would compile with gtk-3 but I'm not sure upstream is there yet.  There is a branch here someone could dig into
https://gitlab.com/sane-project/frontend/xsane/-/branches

**Comment by cuantar** [comment]
Nope, problem is gone!  It installed when I tested now.  Maybe this was related to the portage mirror variable that we changed in our `portage` fork.  Thanks for reminding me.  I think we still should autogen this package; but the original reason for my report is solved.
